src/plugins/about/index.js

```diff
@@ -0,0 +1,133 @@
+const log = require('Debug')('log:about');
+const trace = require('Debug')('trace:about');
+const debug = require('Debug')('debug:about');
+
+const bluebird = require('bluebird');
+const Listener = require('Listener');
+const execFile = require('child_process').execFile;
+//bluebird.promisify(execAsync);
+
+
+//private functions
+
+
+function execAsync(command,args){
```

Comment:
Yea, the bluebird version would give me a pointer to the process instead of the stdout/stderror in the result. Not sure what to do about that so did a quick wrapper.